### PR TITLE
Kamu 142 update and extend cypress e2e tests

### PR DESCRIPTION
**Pull request by dmitriy-borzenko**

### CODE REVIEW

**Comment by zaychenko-sergei** [comment]
/home/sergeiz/kamu/kamu-web-ui-cypress/cypress/e2e/add-event.cy.ts
  99:9  error  It is unsafe to chain further commands that rely on the subject after this command. It is best to split the chain, chaining again from `cy.` in a next command line  cypress/unsafe-to-chain-command

/home/sergeiz/kamu/kamu-web-ui-cypress/cypress/e2e/authentification.cy.ts
  15:9  error  It is unsafe to chain further commands that rely on the subject after this command. It is best to split the chain, chaining again from `cy.` in a next command line  cypress/unsafe-to-chain-command
  18:9  error  It is unsafe to chain further commands that rely on the subject after this command. It is best to split the chain, chaining again from `cy.` in a next command line  cypress/unsafe-to-chain-command

/home/sergeiz/kamu/kamu-web-ui-cypress/cypress/e2e/dataset-page.cy.ts
   53:17  error  It is unsafe to chain further commands that rely on the subject after this command. It is best to split the chain, chaining again from `cy.` in a next command line  cypress/unsafe-to-chain-command
   53:17  error  It is unsafe to chain further commands that rely on the subject after this command. It is best to split the chain, chaining again from `cy.` in a next command line  cypress/unsafe-to-chain-command
   53:17  error  It is unsafe to chain further commands that rely on the subject after this command. It is best to split the chain, chaining again from `cy.` in a next command line  cypress/unsafe-to-chain-command
   72:17  error  It is unsafe to chain further commands that rely on the subject after this command. It is best to split the chain, chaining again from `cy.` in a next command line  cypress/unsafe-to-chain-command
   72:17  error  It is unsafe to chain further commands that rely on the subject after this command. It is best to split the chain, chaining again from `cy.` in a next command line  cypress/unsafe-to-chain-command
   72:17  error  It is unsafe to chain further commands that rely on the subject after this command. It is best to split the chain, chaining again from `cy.` in a next command line  cypress/unsafe-to-chain-command
  131:9   error  It is unsafe to chain further commands that rely on the subject after this command. It is best to split the chain, chaining again from `cy.` in a next command line  cypress/unsafe-to-chain-command

/home/sergeiz/kamu/kamu-web-ui-cypress/cypress/e2e/home-page.cy.ts
  21:9  error  It is unsafe to chain further commands that rely on the subject after this command. It is best to split the chain, chaining again from `cy.` in a next command line  cypress/unsafe-to-chain-command
  33:9  error  It is unsafe to chain further commands that rely on the subject after this command. It is best to split the chain, chaining again from `cy.` in a next command line  cypress/unsafe-to-chain-command

✖ 12 problems (12 errors, 0 warnings)

File: cypress/e2e/authentification.cy.ts [hunk 1]
```diff
@@ -7,23 +7,17 @@ describe("Authentification", () => {
         cy.wait("@graphqlRequest");
     });
 
-    it("Success authentification", () => {
+    xit("Success authentification", () => {
```
Comment (zaychenko-sergei) [inline cypress/e2e/authentification.cy.ts:11]: Please put a comment why test is excluded.

File: cypress/e2e/add-event.cy.ts [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        cy.visit(ProjectLinks.URL_HOME);
+        const date = new Date().getTime();
+        cy.get('[data-test-id="addNewBlock"]').click();
+        cy.get('[data-test-id="dataset-name"]').type(
+            `mock-root-dataset-${date}`,
+        );
+        cy.get('[data-test-id="create-dataset"]').click();
+    });
+
+    xit("Should check add license of the  dataset", () => {
```
Comment (zaychenko-sergei) [inline cypress/e2e/add-event.cy.ts:13]: Please put a comment why test is excluded

**Review by zaychenko-sergei — CHANGES_REQUESTED** [review]
1) Document problems with 2 tests
2) Fix linting errors (see https://docs.cypress.io/guides/core-concepts/retry-ability#Actions-should-be-at-the-end-of-chains-not-the-middle)
3) Set `"printWidth": 120` in Prettier settings

File: cypress/e2e/authentification.cy.ts (hunk 1 above)
Comment (dmitriy-borzenko) [reply cypress/e2e/authentification.cy.ts:11]: Left comment

File: cypress/e2e/add-event.cy.ts (hunk 2 above)
Comment (dmitriy-borzenko) [reply cypress/e2e/add-event.cy.ts:13]: Left comment

**Review by zaychenko-sergei — APPROVED** [review]

**Pull request merged**